compiler/control/OMROptions.cpp

```diff
@@ -1436,6 +1442,17 @@ OMR::Options::setStatic32BitValue(char *option, void *base, TR::OptionTable *ent
    return option;
    }
 
+char *
+OMR::Options::setStaticPercentage(char *option, void *base, TR::OptionTable *entry)
```

Comment:
What is the net effect of this code if a value larger than 100 is provided?
We need to apply some correction and we have two choices: use 100, or use the default value.
What I had in mind was to perform a check in one of the postProcess routines and adjust the value there.